### PR TITLE
gnome-control-center: new aport

### PR DESCRIPTION
**Pull request by Cogitri**
Please do not merge this just yet, the upgrade of gsettings-desktop-schemas
breaks the runtime of at least gnome-desktop.

### CODE REVIEW

**Comment by Cogitri** [comment]
Needs #7258

**Comment by Ikke** [comment]
@Cogitri #7258 has now been merged. Can you prioritize getting this into shape? Thanks

**Comment by Cogitri** [comment]
Needs #7262

File: testing/gnome-control-center/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=gnome-control-center
+pkgver=3.32.1
+pkgrel=1
```
Comment (TBK) [inline testing/gnome-control-center/APKBUILD]:
```suggestion
pkgrel=0
```

Comment (Cogitri) [reply testing/gnome-control-center/APKBUILD]: Ah right, still am used to Void where 1 is the default. Thanks for the pointer!

**Pull request closed without merging**